Maquetado/index.html

```diff
@@ -1,14 +1,17 @@
 <!doctype html>
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
   <title>HTML & CSS Training</title>
   <meta name="description" content="training">
   <meta name="author" content="Wolox">
   <link rel="stylesheet" type="text/css" href="application.css">
 </head>
+
 <body>
-  <h1 class="mainTitle">Título del libro<strong class='bookgenre'> (género)</strong></h1>
+  <img src='./resources/img/book-cover.png' class='bookImage' />
+  <h1 class="mainTitle">Título del libro<strong class='bookGenre'> (género)</strong></h1>
```

Comment:
Let's keep the code easy to read.
```suggestion
  <h1 class="mainTitle">
      Título del libro
      <strong class='bookGenre'> (género)</strong>
  </h1>
```